### DISCUSSION TITLE
Change file name for "Export to file"

### DISCUSSION THREAD

**Discussion by toreriksolem** [Pirate-MIDI/Pirate-MIDI-Features-Bug-Tracking#65] [Editor App Feature Requests] [answered]
I have two devices, and have named them differently in Global > Device Name.

It could be a good thought to have the .json file named "device name + the long string of numbers (if that is necesarry.json"?

MacBook
 OS 11.5.1 
Running web app in Chrome Version 104.0.5112.79
USB-C cable directly to Bridge 6

**Comment by konnsim** [comment]
How does the format "{deviceName} - {dateCreated}.json" sound?
Anything else you would find handy to have in it?

**Reply by toreriksolem** [reply]
> How does the format "{deviceName} - {dateCreated}.json" sound?
> 
> Anything else you would find handy to have in it?

Probably a time stamp aswell, as it's likely I'd make a bunch of exports during a single programming session.

**Comment by konnsim** [comment, marked as answer]
Filename modified to "{deviceName}_{date}_{time}.json" in sorted order.
Eg. "MyBridge6_2022-08-23_13-30-00.json".
